kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/topics/ConnectorsTests.java

```diff
@@ -0,0 +1,96 @@
+        Helpers.INSTANCE.apiHelper.sendMessage(LOCAL, TOPIC_FOR_CONNECTOR,
+                FileUtils.getResourceAsString("message_content_create_topic.json"), " ");
+    }
+
+    @AfterAll
+    @SneakyThrows
+    public static void afterAll() {
+      Helpers.INSTANCE.apiHelper.deleteTopic(LOCAL, TOPIC_FOR_CONNECTOR);
+    }
+
+    @SneakyThrows
+    @DisplayName("should create a connector")
+    @Test
+    void createConnector() {
+        try {
+            pages.openConnectorsList(LOCAL)
+                    .isOnPage()
+                    .clickCreateConnectorButton()
+                    .setConnectorConfig(
+                            SINK_CONNECTOR,
+                            FileUtils.getResourceAsString("config_for_create_connector.json")
+                    )
+                    .connectorIsVisible();
+            pages.openConnectorsList(LOCAL).connectorIsVisibleInList(SINK_CONNECTOR, TOPIC_FOR_CONNECTOR);
+        } finally {
+            Helpers.INSTANCE.apiHelper.deleteConnector(LOCAL, FIRST, SINK_CONNECTOR);
+        }
+    }
+
+    @SneakyThrows
+    @DisplayName("should update a connector")
+    @Test
+    void updateConnector() {
+        try {
+            Helpers.INSTANCE.apiHelper.createTopic(LOCAL, TOPIC_FOR_UPDATE_CONNECTOR);
+            Helpers.INSTANCE.apiHelper.sendMessage(LOCAL, TOPIC_FOR_UPDATE_CONNECTOR,
+                    FileUtils.getResourceAsString("message_content_create_topic.json"), " ");
+            Helpers.INSTANCE.apiHelper.createConnector(LOCAL, FIRST,
+                    CONNECTOR_FOR_UPDATE,
+                    FileUtils.getResourceAsString("config_for_create_connector_via_api.json"));
+            pages.openConnectorsList(LOCAL)
+                    .isOnPage()
+                    .openConnector(CONNECTOR_FOR_UPDATE);
+            pages.openConnectorsView(LOCAL, CONNECTOR_FOR_UPDATE)
+                    .openEditConfig()
+                    .updateConnectorConfig(
+                            FileUtils.getResourceAsString("config_for_update_connector.json"));
+            pages.openConnectorsList(LOCAL).connectorIsUpdatedInList(CONNECTOR_FOR_UPDATE, TOPIC_FOR_UPDATE_CONNECTOR);
+        } finally {
+            Helpers.INSTANCE.apiHelper.deleteConnector(LOCAL, FIRST, CONNECTOR_FOR_UPDATE);
+            Helpers.INSTANCE.apiHelper.deleteTopic(LOCAL, TOPIC_FOR_UPDATE_CONNECTOR);
+        }
+    }
+
+    @SneakyThrows
+    @DisplayName("should delete connector")
+    @Test
+    void deleteConnector() {
+        Helpers.INSTANCE.apiHelper.createTopic(LOCAL, TOPIC_FOR_DELETE_CONNECTOR);
+        Helpers.INSTANCE.apiHelper.sendMessage(LOCAL, TOPIC_FOR_DELETE_CONNECTOR,
+                FileUtils.getResourceAsString("message_content_create_topic.json.json"), " ");
+        Helpers.INSTANCE.apiHelper.createConnector(LOCAL, FIRST,
+                CONNECTOR_FOR_DELETE,
+                FileUtils.getResourceAsString("delete_connector_config.json"));
```

Comment:
can we move all prepartion in `beforeAll` ? (in both update/delete connector) so the body of test method would contain  the most relevant details